library/time/to_time_spec.rb

```diff
@@ -0,0 +1,16 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'time'
+
+ruby_version_is "2.4" do
+  describe "Time#to_time" do
+    it "returns itself in the same timezone" do
+      with_timezone("America/Regina") do
+        time = Time.new(2012, 2, 21, 10, 11, 12)
+        time.to_time.should == time
```

Comment:
This is basically the same test as below, did you intend to build the `time` outside the timezone and verify it's the same even if TZ changes?